python/raft-dask/setup.cfg

```
@@ -46,4 +46,4 @@ skip=
 
 [options]
 packages = find:
-python_requires = >=3.7,<3.10
+python_requires = >=3.7
```

Oops. Meant to update this in https://github.com/rapidsai/raft/pull/1120. I missed it because this is using both `setup.cfg` and `setup.py`. I didn't think of that possibility -- I only updated `setup.py` and `pyproject.toml` files in the CUDA 11.8 / Python 3.10 updates for conda packages.

This suggestion would align with the rest of RAPIDS.
```suggestion
python_requires = >=3.8,<3.11
```